docs/02.API-REFERENCE.md

```diff
@@ -243,7 +243,7 @@ Option bits for [jerry_parse_options_t](#jerry_parse_options_t).
  - JERRY_PARSE_NO_OPTS - No options passed
  - JERRY_PARSE_STRICT_MODE - Enable strict mode
  - JERRY_PARSE_MODULE - Parse source as an ECMAScript module
- - JERRY_PARSE_HAS_ARGUMENT_LIST - `argument_list` field is valid
+ - JERRY_PARSE_HAS_ARGUMENT_LIST - `argument_list` field is valid, this also means that function parsing will be done
```

Comment:
Update the comment in the include/types.h as well (these kept in sync).

Reply:
Good idea. I'll update the related header file.